### ISSUE TITLE
How are the indicators obtained

### ISSUE THREAD

**Issue by Scw1217** [MIC-DKFZ/nnDetection#303]
Hello, I am currently using your framework to study pancreatic tumor detection. I have trained a five - fold model and made predictions. After prediction, I generated the coordinates of the detection boxes. I would like to ask how the indicators AP@0.1 and AP@0.5 are obtained. Regarding the AP@0.1 on the validation set for the pancreas, which reached 0.79, how is this indicator obtained?

**Comment by andres2631996** [comment]
Hello, AP stands for Average Precision, which measures the area under the precision-recall curve. Both precision and recall need the computation of true positives. AP@0.1 considers true positives those predicted boxes with at least an IoU of 0.1 with the ground-truth boxes, while AP@0.5 considers those predicted boxes with at least an IoU of 0.5 with the ground-truth boxes. The higher the AP, the better
